tools/tf2openapi/tfmodel/tftensor.go

```diff
@@ -0,0 +1,68 @@
+package tfmodel
+
+type TFTensor struct {
```

Comment:
Can you add a comment to each of these fields? Perhaps a link as well to the tensorflow deriving this in a comment?